runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpStateSingle.java

```diff
@@ -163,6 +253,17 @@ else if (durationHours > 0)
 		return String.format("%02d:%02d", durationMinutes, durationSeconds);
 	}
 
+	private static Map<String, Integer> loadNpcHealth()
```

Comment:
If npc health is going to be required for >1 plugin it should be moved somewhere central (`net.runelite.client.game`?) - or perhaps exposed via a service in the oppoinfo plugin.